### ISSUE TITLE
Allow html

### ISSUE THREAD

**Issue by Fresheyeball** [atom/autocomplete-plus#599]
https://github.com/edubkendo/atom-elm/issues/23 

I realize that letting in HTML is a security risk, but sanitizing HTML for security is kind of a known path. I suggest that HTML be allowed in after passing through a sanitization step. Failing this step could throw an error in developer mode.

**Comment by icetee** [comment]
Yes, no parse HTML. But the [atom-ide-ui](https://atom.io/packages/atom-ide-ui) rendered HTML tags. It would be a great feature, I miss it.

<img width="513" alt="screenshot 2018-05-09 16 18 42" src="https://user-images.githubusercontent.com/1855340/39820124-5fac633a-53a5-11e8-9606-cfea2cfe8bcc.png">

UI:
<img width="676" alt="screenshot 2018-05-09 16 27 09" src="https://user-images.githubusercontent.com/1855340/39820372-f4beebf0-53a5-11e8-9b42-98c2c9083a48.png">
